hw06/series.py

```diff
@@ -0,0 +1,70 @@
+
+
+# write Fibonacci series up to n
+def fibonacci(n):
+    """The fibonacci function returns the nth values of the
+    fibonacci series where n is a user-input number."""
+    # create an array to populate
+    fib = list(range(1, n + 2))
+    # assign the first two values in the array
+    fib[0] = 0
```

Comment:
Rather than create the list above with all those numbers we're not using, what if we just said `fib = [0,1]`